### DISCUSSION TITLE
Jamulus & Yamaha P515 Keyboard Sampling Rates

### DISCUSSION THREAD

**Discussion by ylama-57** [jamulussoftware/jamulus#1564] [Technical Support & Help]
Hi. I'm trying to connect my Yamaha P515 keyboard to my laptop thru' USB cable in order to use Jamulus for live music jamming. I've downloaded Yamaha Steinburg USB ASIO driver. When I connect my Yamaha to my laptop, the laptop recognizes it. When I double-click Jamulus, I get the following message:
"No usable ASIO audio device (driver) found. In the following there is a list of all available drivers with the associated error message: Yamaha Steinburg USB ASIO : The audio device does not support the required sample rate. The required sample rate is : 48000 Hz".
I checked the sampling rate on my Yamaha, it is fixed at 44100 Hz and cannot be changed. 
It asks me "do you to change the ASIO driver settings?". I click "Yes". It opens Yamaha Steinberg USB Driver window and shows Device: Digital Piano, Mode: Standard, Buffer Size: 256 samples and under Digital Piano tab, it shows Sampling Rate 44100 Hz. I cannot change that.
I'd appreciate some solution to this issue. Thanks.

**Comment by dcorson-ticino-com** [comment]
I think the solution, if there is one, will have to come from Yamaha.
Jamulus does not support sample rates other than 48kHz, sorry to say.

**Reply by hoffie** [reply]
I'm wondering if ASIO4ALL is worth a try here?

**Reply by ylama-57** [reply]
I tried ASIO4ALL as well - same issue !

**Reply by hoffie** [reply]
Including the different options?
Especially the resampling one sounds promising (https://www.image-line.com/fl-studio-learning/fl-studio-online-manual/html/envsettings_asio4all.htm Ctrl+F *resample*).

Do you get the same error message with ASIO4ALL?
